src/test/java/org/pih/petl/job/SqlServerImportJobTest.java

```diff
@@ -28,12 +29,35 @@
         SpringRunnerTest.setupEnvironment();
     }
 
+    @After
+    public void dropTablesInTargetDB() throws Exception{
```

Comment:
I tried to fiddle around with dirtying the context, etc, but couldn't get that to work, so just wrote this to drop the target table after each run.